crates/ark/src/modules/rstudio/dialogs.R

```diff
@@ -11,3 +11,12 @@
 .rs.api.showQuestion <- function(title, message, ok = NULL, cancel = NULL) {
     .ps.ui.showQuestion(title, message, ok, cancel)
 }
+
+
+#' @export
+.rs.api.showPrompt <- function(title, message, default) {
+    # rstudioapi doesn't pass the timeout directly but sets it as
+    # `rstudioapi.remote.timeout` option
```

Comment:
The actual R function does pass it in as plain old `60`, so I think we need to do some kind of coercion here to integer.

If I run the function without specifying `timeout`, I see an error:

```
Unexpected vector type (expected integer; got double)
```

If I pass in something like `timeout = 40L`, then I see the new dialog working:

<img width="1374" height="1044" alt="Screenshot 2025-09-12 at 8 08 42 PM" src="https://github.com/user-attachments/assets/f022fa4f-35e4-4b6a-931c-0d2c3d7b0913" />

Reply:
You're right, thanks for catching that! Addressed in https://github.com/posit-dev/ark/pull/920/commits/eded84af802f21e70a2d0cad526f97b1ddb80e61.